non-free.md

```diff
@@ -17,7 +17,6 @@
 ## Content Management Systems (CMS)
 
 - [CraftCMS](https://craftcms.com/) `⊘ Proprietary` - Craft is a content-first CMS that aims to make life enjoyable for developers and content managers alike. ([Demo](https://demo.craftcms.com/)) `PHP`
-- [Gazelle](https://github.com/WhatCD/Gazelle) - Gazelle is a web framework geared towards private BitTorrent trackers. Although naturally focusing on music, it can be modified for most needs. `unlicensed` `PHP`
```

Comment:
it was the last software in this category (CMS for torrent trackers), for alternatives check https://github.com/HDVinnie/Torrent-Tracker-Platforms. Currently active projects include:
- https://github.com/HDInnovations/UNIT3D-Community-Edition
- https://github.com/Mosasauroidea/GazellePW
- https://github.com/xiaomlove/nexusphp
- https://github.com/torrust/torrust-index